st2common/st2common/models/db/execution.py

```diff
@@ -71,9 +71,7 @@ class ActionExecutionDB(stormbase.StormFoundationDB):
             {'fields': ['end_timestamp']},
             {'fields': ['status']},
             {'fields': ['parent']},
-            {'fields': ['-start_timestamp', 'action']},
-            {'fields': ['-start_timestamp', 'parent']},
-            {'fields': ['-start_timestamp', 'action', 'parent']}
```

Comment:
So at this point, I think we should add them when there's obvious performance problem. Otherwise, it may have negative effect having too many indexes.